source/_UIAHandler.py

```diff
@@ -355,7 +355,10 @@ def _isBadUIAWindowClassName(self, windowClass):
 		"Given a windowClassName, returns True if this is a known problematic UIA implementation."
 		# #7497: Windows 10 Fall Creators Update has an incomplete UIA implementation for console windows, therefore for now we should ignore it.
 		# It does not implement caret/selection, and probably has no new text events.

```

Comment:
I think you can remove these comments here and move this information to the doc string of `shouldUseUIAConsole`.